src/plugins/data/public/ui/search_bar/search_bar.tsx

```diff
@@ -28,27 +28,25 @@
  * under the License.
  */
 
-import { compact } from 'lodash';
 import { InjectedIntl, injectI18n } from '@osd/i18n/react';
 import classNames from 'classnames';
+import { compact, get, isEqual } from 'lodash';
```

Comment:
nit: i think we have the isEqual `lodash` replaced. if you mind doing a quick check? but we had a goal to clean up lodash usage:

https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3660
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3899

but i would do this only if you have time. i'm guilty of adding more lodash as well.

Reply:
the PR didn't touch lodash usage, this came from organizing imports of the previous version of search_bar.tsx, which had them imported on separate lines
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/a4aa682226b9d22d2c6e7a41a11a44225b6d7db1/src/plugins/data/public/ui/search_bar/search_bar.tsx#L31-L36

it might be better to do this in another PR so that the change is more isolated